crates/uv/src/commands/auth/dir.rs

```diff
@@ -0,0 +1,13 @@
+use anstream::println;
+use owo_colors::OwoColorize;
+
+use uv_auth::{TextCredentialStore};
+use uv_fs::Simplified;
+
+/// Show the credentials directory.
+pub(crate) fn dir() -> anyhow::Result<()> {
+    let root = TextCredentialStore::directory_path()?;
+    println!("{}", root.simplified_display().cyan());
```

Comment:
We could warn if they're not using the plaintext backend, but... IDK if it's helpful.